system/netdata.logrotate.in

```diff
@@ -7,6 +7,6 @@
 	notifempty
 	sharedscripts
 	postrotate
-		/bin/kill -HUP `cat @localstatedir_POST@/run/netdata/netdata.pid 2>/dev/null` 2>/dev/null || true
+		/usr/sbin/netdatacli reopen-logs
```

Comment:
This needs to use templating to point to the right location (instead of `/usr/sbin/netdatacli`, it should be `@sbindir_POST@/netdatacli`), and it also needs to account for the possibility of the agent not actually running (because if the agent isn’t running, `netdatacli` will fail).